akvo/rsr/models/report.py

```diff
@@ -39,6 +39,7 @@ class Report(models.Model):
         help_text=_(u'Enter the parametrized path for downloading the report'),
     )
     formats = models.ManyToManyField(ReportFormat)
+    organisations = models.ManyToManyField('Organisation')
```

Comment:
This field needs to be null=True, blank=True otherwise it's not possible to save an instance without any org